### PR TITLE
New layout

### PR DESCRIPTION
**Pull request by LeonardoFerreiraa**
new layout for `vacancies/index` and `vacancies/new`

extra seeds

### CODE REVIEW

**Comment by duduribeiro** [comment]
@cristofersousa can u look on this please?

**Comment by duduribeiro** [comment]
@LeonardoFerreiraa can u provide some screenshots of the difference between the old version and the new one?

File: app/views/vacancies/_vacancy.html.erb [hunk 1]
```diff
@@ -1,7 +1,15 @@
-<li class="vacancy-item">
-  <h3>
+<section class="vacancy-item">
```
Comment (duduribeiro) [inline app/views/vacancies/_vacancy.html.erb]: For me this is incorrect. IMO, `section` should only appear once in a HTML document. What you think @cristofersousa ?

**Comment by LeonardoFerreiraa** [comment]
The new layout screenshots.

pasted image at 2017_03_02 07_20 am 1
pasted image at 2017_03_02 07_20 am
pasted image at 2017_03_02 07_21 am
pasted image at 2017_03_02 07_22 am 1
pasted image at 2017_03_02 07_22 am

**Pull request merged**